### ISSUE TITLE
ignores errors

### ISSUE THREAD

**Issue by jelmer** [namaggarwal/splitwise#22]
The API for e.g. create_expense returns a list of created expenses as well as a list of errors. The list of errors is currently ignored.

**Comment by lefterav** [comment]
Additionally, when an error occurs while creating an expense, the error given by python in non-informative:


```
File "__init__.py", line 251, in createExpense

expense = Expense(content["expenses"][0])
IndexError: list index out of range
```

**Comment by namaggarwal** [comment]
Library does not handle the errors well at the moment. I will enhance this. This will be a breaking change so I will release a major version update to this

**Comment by namaggarwal** [comment]
This is fixed since version 2.0.0
